### PR TITLE
fix: prevent dialog input autofocus

### PR DESCRIPTION
**Pull request by jose-costa-frontify**
CU-8695mu8rt

### CODE REVIEW

File: packages/components/src/components/Dialog/Dialog.stories.tsx [hunk 1]
```diff
@@ -313,6 +314,29 @@ export const WithCustomElementAnnouncements: Story = {
     },
 };
 
+export const WithFocusableContent: Story = {
+    args: {
+        children: 'Hello World',
+    },
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
```
Comment (SamuelAlev) [inline packages/components/src/components/Dialog/Dialog.stories.tsx:323]: Dialog component not Flyout

File: packages/components/src/utilities/domUtilities.ts [hunk 2]
```diff
@@ -75,3 +75,38 @@ export function syncRefs(localRef: RefObject<HTMLDivElement>, forwardedRef: Forw
...
 }
+
+/**
+ * Sets the 'autoFocusVisible' data attribute to 'false' on the current target of a mouse event.
+ * This function is typically used to indicate that an element has been interacted with via mouse,
+ * which should prevent the focus ring from being displayed when the element receives focus.
+ *
+ * @param mouseEvent - The mouse event object.
+ */
+export function addAutoFocusAttribute(mouseEvent: ReactMouseEvent<HTMLButtonElement, MouseEvent>) {
```
Comment (SamuelAlev) [inline packages/components/src/utilities/domUtilities.ts:86]: Can be added to Flyout component too

Comment (jose-costa-frontify) [reply packages/components/src/utilities/domUtilities.ts:86]: yep I did!

File: packages/components/src/components/Dialog/Dialog.stories.tsx (hunk 1 above)
Comment (jose-costa-frontify) [reply packages/components/src/components/Dialog/Dialog.stories.tsx:323]: no more pasta for me today

**Pull request merged**